add a note not to apply encrypted secrets directly (#506)
fluxcd/flux2: docs/guides/mozilla-sops.md

You can now commit the encrypted secret to your Git repository.

!!! hint

>     Note that you must not apply encrypted secrets directly onto the cluster or you will get the yaml invalidation error. SOPS encrypted secrets are designed to be consumed by the Kustomization controller. Please see the mechanism at the end of this document.

Reviewer: ```suggestion
    Note that you shouldn't apply the encrypted secrets onto the cluster with kubectl. SOPS encrypted secrets are designed to be consumed by kustomize-controller. 
```
